tests/main/snap-userd-reexec/task.yaml

```diff
@@ -16,6 +16,11 @@ restore: |
     fi
 
 execute: |
+    if [ "$MODIFY_CORE_SNAP_FOR_REEXEC" = 0 ]; then
```

Comment:
@sergiocazzolato we have more systems that re-exec other than just ubuntu-18* so we should update this test to run more places

Reply:
sure, I'll do it in a following pr